test/libsolidity/GasCosts.cpp

```diff
@@ -99,30 +99,30 @@ BOOST_AUTO_TEST_CASE(string_storage)
 	auto evmVersion = dev::test::Options::get().evmVersion();
 
 	if (evmVersion <= EVMVersion::byzantium())
-		CHECK_DEPLOY_GAS(134145, 130831, evmVersion);
+		CHECK_DEPLOY_GAS(134209, 130895, evmVersion);
```

Comment:
Do we know why gas costs increased despite no code changes in this PR? It seems a bit random.

Reply:
Just followed the thread in this PR, and it seems we have a good hypothesis :-) Please ignore my comment.

Reply:
At some point, we should also make this automatic, I think.